libs/designsystem/list/src/list.component.integration.spec.ts

```diff
@@ -129,6 +129,62 @@ describe('ListComponent', () => {
         });
       });
     });
+
+    describe('with shape="none"', () => {
+      beforeEach(() => {
+        spectator.setInput('shape', 'none');
+        spectator.detectChanges();
+      });
+      it('should apply transparent background to kirby-list-items', () => {
+        const ionItem = spectator.queryAll('ion-item');
+        expect(ionItem.length).toBeGreaterThan(0);
+        ionItem.forEach((item) => {
+          const itemNative = item.shadowRoot.querySelector('.item-native');
+          expect(itemNative).toHaveComputedStyle({ 'background-color': 'rgba(0, 0, 0, 0)' });
+        });
+      });
+      it('should apply "medium" border color to all but last item when not in card', () => {
+        const ionItemSlidingList = spectator.queryAll('ion-item-sliding');
+        const lastIonItemSliding = ionItemSlidingList[ionItemSlidingList.length - 1];
+
+        expect(ionItemSlidingList.length).toBeGreaterThan(0);
+        ionItemSlidingList.forEach((item) => {
+          if (item !== lastIonItemSliding) {
+            expect(item).toHaveComputedStyle({ 'border-bottom-color': getColor('medium') });
+          }
+        });
+      });
+    });
+  });
+
+  describe('with list inside card and shape="none"', () => {
+    beforeEach(async () => {
+      spectator = createHost<ListComponent>(
+        `
+        <kirby-card>
+          <kirby-list [items]="[{ name: 'Item1' }, { name: 'Item2' }, { name: 'Item3' }]" (itemSelect)="($event)" shape="none">
```

Comment:
Do you need to listen for `itemSelect`? 🤔 
```suggestion
          <kirby-list [items]="[{ name: 'Item1' }, { name: 'Item2' }, { name: 'Item3' }]" shape="none">
```